types/jsonwebtoken/index.d.ts

```diff
@@ -77,6 +78,19 @@ export interface DecodeOptions {
     complete?: boolean;
     json?: boolean;
 }
+
+export type SecretOrPublicKeyCallback = {
+    (
+        err: VerifyErrors,
+        secretOrPublicKey: string | Buffer
+    ): void;
+}
+export type GetSecretOrPublicKeyCallback = {
+    (
+        header: any,
```

Comment:
Sorry to drag this out but `any` is not very precise, this is always going to be an `object`, you could even go as far as documenting the header here as: `export interface Header { alg: string, typ?: string }`.

Reply:
I can add that. I'm just concerned about not having all of the possible JWT header claims here. If I could find an exhaustive list, I could add them all.